src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -95,6 +95,12 @@ public async Task<IEnumerable<Type>> GetExtensionsStartupTypesAsync()
                     _logger.ScriptStartUpErrorLoadingExtensionBundle();
                     return new Type[0];
                 }
+
+                if (SystemEnvironment.Instance.IsPlaceholderModeEnabled())
+                {
+                    // Calling this log statement in the placeholder mode to avoid jitting during specializtion
+                    _logger.ScriptStartNotLoadingExtensionBundle(extensionsPath, bundleConfigured, isPrecompiledFunctionApp, isLegacyExtensionBundle);
```

Comment:
Can we do something to make it super-obvious that this isn't an accurate log? Just thinking:

```
_logger.ScriptStartNotLoadingExtensionBundle("WARMUP_LOG_ONLY", bundleConfigured, isPrecompiledFunctionApp, isLegacyExtensionBundle);
```

Reply:
done